.github/workflows/unit-test.yml

```diff
@@ -0,0 +1,30 @@
+name: Unit-test
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - '**'
+
+jobs:
+  jest:
```

Comment:
```suggestion
  test:
```
Is this typo?

Reply:
Ah! Thank you so much.
It's a mistake.
I copied this from another project and forgot to change this job name. :p